build.rs

```diff
@@ -8,24 +10,30 @@ macro_rules! log {
 }
 
 fn main() {
-    // Build and link IOHK libsodium
-    run("git", |command| {
-        command
-            .arg("submodule")
-            .arg("update")
-            .arg("--init")
-            .arg("--recursive")
-            .arg("--force")
-    });
-
-    // Build libsodium automatically (as part of rust build)
     #[cfg(not(feature = "libsodium-sys"))]
     {
-        let libsodium = autotools::Config::new("contrib/libsodium/").reconf("-vfi").build();
-        println!("cargo:rustc-link-search=native={}", libsodium.join("lib").display());
-        println!("cargo:rustc-link-lib=static=sodium");
+        // Use set libsodium env path
```

Comment:
This code is failing some of the formatting checks. Check the main page and run all the commands locally to make sure it will pass the code format validation checks.

https://github.com/cardano-community/cncli

Reply:
Sounds good, will do. What about removing the addition of /lib after the env variable? Do you think it makes sense to do that?  so people actually point to the lib directory?

Reply:
yes. I would remove this.

Reply:
Sorry for the delay. i'm just now testing the fix with the port as well, to make it's actually building.